services/reventService.tsx

```diff
@@ -438,6 +438,13 @@ export async function getContestList() {
   //   return fetchEndpoint('contests/', GET)
 }
 
+export async function getContestById(id) {
+  const contests = await getContestList()
+  const contest = contests.find((contest) => contest.id === id)
+
+  return contest
+}
```

Comment:
you should create the service for `getContest` since you already have the gql query for it. And use it here instead of doing a filter on the list

Reply:
not beeing used, will delete it